src/GraphQL/GraphQLBuilderExtensions.cs

```diff
@@ -1077,6 +1093,21 @@ public static IGraphQLBuilder AddMetrics(this IGraphQLBuilder builder, Func<Exec
             });
             return builder;
         }
+
+        /// <summary>
+        /// Configures the execution to add Apollo Tracing results to the returned <see cref="ExecutionResult"/> instance.
+        /// </summary>
+        public static IGraphQLBuilder AddApolloTracingResults(this IGraphQLBuilder builder)
```

Comment:
```suggestion
        public static IGraphQLBuilder AddApolloTracing(this IGraphQLBuilder builder)
```
?

Reply:
Also I think that all `AddMetrics` methods may and should be absorbed by this new `AddApolloTracing` method.

Reply:
#3125